lib/msf/core/exploit/remote/mssql.rb

```diff
@@ -46,6 +46,10 @@ def initialize(info = {})
     register_autofilter_services(%W{ ms-sql-s ms-sql2000 sybase })
   end
 
+  def set_session(client)
```

Comment:
We'll probably want a similar pattern to https://github.com/rapid7/metasploit-framework/pull/18759/files#r1479720603

Reply:
Probably makes sense to add this here. I think adding a status for the user to let them know when they're using a session to run a module could be good:
```
print_status("Using existing session #{session.sid}")
```